htdocs/fourn/facture/card.php

```diff
@@ -2411,6 +2411,8 @@
 				print $langs->trans('AlreadyPaid');
 			print ' :</td><td align="right"'.(($totalpaye > 0)?' class="amountalreadypaid"':'').'>' . price($totalpaye) . '</td><td>&nbsp;</td></tr>';
 
+            $totalpaye = $object->getSommePaiement();
```

Comment:
The line just before use $totalpaye. I think this line should be moved up, shouldn't it ?